components/landing/LandingCta.vue

```diff
@@ -75,5 +78,16 @@ export default class extends Vue {
     margin-left: $spacing-06;
     fill: currentColor;
   }
+
+  svg {
+      overflow: visible;
+  }
+  path:nth-child(1) {
+    transform: translate(0, 0);
+    transition: transform 0.2s ease-in-out;
+  }
+  &:hover path:nth-child(1) {
+    transform: translate(3px, 0);
```

Comment:
@korgan00 - I _think_ this arrow shift was going to be `8px`, if I remember correctly... 

@JRussellHuffman can verify.

Reply:
8px may be a bit much compared to 3px. Can we do 4px?
Its tiny, but generally the IBM Design standards push for everything to be in intervals of 8 (which is where that first number came from). In this case, we can do 1/2 of 8, so 4px is fine.

For the "launch" icon (eg. on the learn page), is it translating the same distance?